src/megaapi_impl.cpp

```diff
@@ -12021,6 +12030,15 @@ void MegaApiImpl::getemaillink_result(error e)
     fireOnRequestFinish(request, MegaError(e));
 }
 
+void MegaApiImpl::resendverificationemail_result(error e)
+{
+    if (requestMap.find(client->restag) == requestMap.end()) return;
+    MegaRequestPrivate *request = requestMap.at(client->restag);
```

Comment:
There is a new way to avoid multiple lookups in the `requestMap`. Please, keep the iterator returned by `find()` and use it to get the pointer to the `request`.